src/Lifecycle.ts

```diff
@@ -71,13 +71,22 @@ import { persistOidcAuthenticatedSettings } from "./utils/oidc/persistOidcSettin
 const HOMESERVER_URL_KEY = "mx_hs_url";
 const ID_SERVER_URL_KEY = "mx_is_url";
 
+/**
+ * Used as storage key
+ */
 const ACCESS_TOKEN_STORAGE_KEY = "mx_access_token";
 const REFRESH_TOKEN_STORAGE_KEY = "mx_refresh_token";
 /**
- * Used during encryption/decryption of token
+ * Used as initialization vector during encryption in persistTokenInStorage
+ * And decryption in restoreFromLocalStorage
  */
 const ACCESS_TOKEN_NAME = "access_token";
 const REFRESH_TOKEN_NAME = "refresh_token";
```

Comment:
can we call them `ACCESS_TOKEN_IV` and `REFRESH_TOKEN_IV` to better reflect their purpose?